tst/teststandard/ctblmono.tst

```diff
@@ -0,0 +1,31 @@
+gap> START_TEST("ctblmono.tst");
+
+#
+# the following test comes from https://github.com/gap-system/gap/issues/4452
+# this used to give method not found for non-solvable groups
+#
+gap> IsMinimalNonmonomial(SL(2,3));
+true
+gap> IsMinimalNonmonomial(GL(2,3));
+false
+gap> IsMinimalNonmonomial(PSL(2,4));
+true
+gap> IsMinimalNonmonomial(PGL(2,5));
+false
+gap> IsMinimalNonmonomial(PSL(2,7));
+false
+gap> IsMinimalNonmonomial(PSL(2,8));
+true
+gap> IsMinimalNonmonomial(PSL(2,9));
+false
+gap> IsMinimalNonmonomial(PSL(2,11));
+false
+gap> IsMinimalNonmonomial(PSL(2,27));
+true
+gap> IsMinimalNonmonomial(PSL(3,3));
+false
+gap> IsMinimalNonmonomial(Sz(8));
+true
```

Comment:
For full code coverage, it would be nice to also test the negative case here
```suggestion
gap> IsMinimalNonmonomial(Sz(8));
true
gap> IsMinimalNonmonomial(Sz(32));
false
```

Reply:
I think Sz(2^9) is the first non-monomial (its only crime is that it contains Sz(2^3) as a subgroup). I'll check if GAP can handle such a large group.

Reply:
I think this is now handled, though a bit awkwardly due to the size of the group. I'll submit a separate pull request that helps suzuki groups know they are not solvable.